app/mailers/application_mailer.rb

```diff
@@ -15,6 +15,8 @@ def user_email_with_name(user)
   end
 
   def mail_to_user(user, subject, **options)
+    return if user.email_status_invalid?
```

Comment:
This would be better in `app/commands/user/send_email.rb` alongside `users.noreply.github.com` as that will correctly set the status of the email to `skipped` not `sent`.

Reply:
Ah, that's much better. I'm still fairly new to the email sending and how that works.